docs/developer-docs/latest/developer-resources/database-apis-reference/rest/filtering-locale-publication.md

```diff
@@ -0,0 +1,379 @@
+---
+title: Filtering for REST API - Strapi Developer Docs
+description: Use Strapi's REST API to filter the results of your requests.
+sidebarDepth: 3
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest-api/filtering-locale-publication.html
+---
+
+# REST API: Filtering, Locale, and Publication State
+
+The [REST API](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md) offers the ability to filter results found with its ["Get entries"](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md#get-entries) method. If the [Internationalization (i18n) plugin](/developer-docs/latest/plugins/i18n.md) is enabled on your content-type it's possible to filter by locale and likewise if you have [Draft & Publish](/developer-docs/latest/concepts/draft-and-publish.md) enabled it's possible to filter based on a `live` or `draft` state.
```

Comment:
```suggestion
The [REST API](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md) offers the ability to filter results found with its ["Get entries"](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md#get-entries) method.<br/>
Using optional Strapi features can provide some more filters:

* If the [Internationalization (i18n) plugin](/developer-docs/latest/plugins/i18n.md) is enabled on a content-type, it's possible to filter by locale.
* If the [Draft & Publish](/developer-docs/latest/concepts/draft-and-publish.md)is  enabled, it's possible to filter based on a `live` or `draft` state.
```

Cleaner look (using lists) when we have multiple possibilities + avoid using "you" (neutral tone, see our Handbook > 12 rules)